subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync.c

```diff
@@ -474,7 +474,8 @@ static int is_abort_cb(void *next, void *curr, lll_prepare_cb_t *resume_cb)
 			struct lll_scan_aux *lll_aux;
 
 			lll_aux = ull_scan_aux_lll_is_valid_get(next);
-			if (!lll_aux) {
+			if (IS_ENABLED(CONFIG_BT_CTLR_SYNC_PERIODIC_SKIP_ON_SCAN_AUX) ||
+			    !lll_aux) {
```

Comment:
What is the reason why we do not want this to be the (unconfigurable) default? What is the use case for disabling `BT_CTLR_SYNC_PERIODIC_SKIP_ON_SCAN_AUX`?

Reply:
Update the Kconfig help with the answer.

Reply:
Just to make sure I understand what is happening: Enabling this Kconfig will basically prioritize scan AUX events over periodic sync events? 

If that is the case, would it make sense, for a longer term plan, to define a set of priorities for the events, so you can basically do

```c
CONFIG_SYNC_PERIODIC_PRIO=10
CONFIG_SCAN_AUX_PRIO=5
```

or 

```c
CONFIG_SYNC_PERIODIC_PRIO=10
CONFIG_SCAN_AUX_PRIO=15
```

To control the prioritizes? Are there any other events (e.g. ACL connections events or ISO sync receiver events) that may also be impacted?

Reply:
> Just to make sure I understand what is happening: Enabling this Kconfig will basically prioritize scan AUX events over periodic sync events?

No, I have carefully used the term permit. Zephyr Controller does not use priorities for state/roles, rather the preference is to use round-robin, skip count, age and just-in-time negotiation to determine a fair scheduling result.

I have not yet concluded on a scheduling scheme between auxiliary PDU scan in unreserved time space and other states/roles. For the Periodic Sync event in unreserved time space that is overlapping with other Periodic Sync events, a round-robin scheduling is implemented in https://github.com/zephyrproject-rtos/zephyr/pull/49579. Adding  an age factor would be better, would consider it in the future.

> Are there any other events (e.g. ACL connections events or ISO sync receiver events) that may also be impacted?

A similar implementation will be added for ISO sync in the future. Use of priorities, if not by use of dynamic priorities at runtime will lead to starvation/livelock of certain state/roles.

ACL connections use reserved time space hence use round-robin and being under supervision timeout as the sole conditions of mitigating overlaps.